plugins/axios.js

```diff
@@ -20,6 +20,13 @@ function renameFields(params) {
   return params
 }
 
+function filterFunc(prefix, value) {
+  if (value === '') {
+    return
+  }
+  return value
+}
+
 export default function ({ $axios, store, redirect }, inject) {
   const ztfApi = $axios.create({
     baseURL: 'http://dev.api.alerce.online/',
```

Comment:
We must change this static api. Extract url of api from dot env files

Reply:
@dirodriguezm is this another issue?

Reply:
Yes that should be in another issue. I will add it now